pkg/build/addimage/defaults.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addimage
+
+// DefaultImage is the default name:tag for the built image
+const DefaultImage = "kindest/custom-node:latest"
+
+// DefaultBaseImage is the default base image used
+const DefaultBaseImage = "docker.io/kindest/node:latest"
```

Comment:
do we want to use different defaults than nodeimage https://github.com/kubernetes-sigs/kind/blob/9fe964443fe34eb3bb59080d513eadeaa35370cf/pkg/build/nodeimage/defaults.go#L23

Reply:
I will push a change for this.

Reply:
Now, I am using the default node image that gets built.

https://github.com/kubernetes-sigs/kind/blob/e3864c0be8cc3b17380d7d98d7f2632212ef79b6/pkg/apis/config/defaults/image.go#L21